Documentation/FAQs.md

```diff
@@ -1,6 +1,13 @@
 # MessageKit Frequently Asked Questions
 
-**Why doesn't the `MessageInputBar` appear in my controller?**
```

Comment:
Thanks for updating the FAQ 💯

Reply:
I souldn't be credit for updating FAQ, I had no ideea why FAQ had been updated in my PR.